internal/provider/resource_tfe_registry_module_test.go

```diff
@@ -361,7 +361,7 @@ func TestAccTFERegistryModule_noCodeModule(t *testing.T) {
 		RegistryName: tfe.PublicRegistry,
 		Namespace:    "terraform-aws-modules",
 		Organization: &tfe.Organization{Name: orgName},
-		NoCode:       true,
+		NoCode:       false, // changes to false on refresh
```

Comment:
can you add more context to this comment? Was this changed to `false` because the `tfe_no_code_module` block has been added to the config for this test?

Reply:
I removed that comment. The reason the value is now false is because we're no longer asking the `tfe_registry_module` to set the `no-code` status. That is now set by the `tfe_no_code_module` resource